upgrade/1.2.1/upgrade_network.md

```diff
@@ -0,0 +1,102 @@
+# Upgrade and validate Switch Configurations
+
+## Prerequisites
+
+* SSH access to the switches or the running configuration file
+* Upgrade CANU: [Install/Upgrade CANU](../../operations/network/management_network/canu_install_update.md)
+  * Run `canu --version` to see version
+  * CANU installed with version 1.6.13 or greater.
+* Use the paddle file (CCJ) or validate SHCD [Validate SHCD](../../operations/network/management_network/validate_shcd.md)
+* System Layout Service (SLS) input file. [Collect Data](../../operations/network/management_network/collect_data.md)
+
+> **CAUTION:** All of these steps should be done using an out-of-band connection. This process is disruptive and will require downtime.
+
+## Generate Configuration Files
+
+Ensure that the correct architecture (`-a` parameter) is selected for the setup in use.
+
+The following are the different architectures that can be specified:
+
+* `Tds` – Aruba-based Test and Development System. These are small systems characterized by Kubernetes NCNs cabled directly to the spine.
+* `Full` – Aruba-based Leaf-Spine systems. These are usually customer production systems.
+* `V1` – Dell and Mellanox based systems of either a TDS or Full layout.
+
+Generating a configuration file can be done for a single switch, or for the full system. Below are example commands for both scenarios:
+
+**Important:** Modify the following items in your command:
+
+* `--csm` : Which CSM version configuration do you want to use? For example, `1.2` or `1.0`
+* `--a`   : What is the system architecture? (See above)
+* `--ccj` : Match the `ccj.json` file to the one you created for your system.
+* `--sls` : Match the `sls_file.json` to the one you created for your system.
+* `--custom-config` : Pass in a switch configuration file that CANU will inject into the generated configuration. For more information, see the [CANU documentation](https://github.com/Cray-HPE/canu#generate-switch-configs-including-custom-configurations).
+
+* Generate configuration files for full system:
+
+    ```console
+    ncn# canu generate network config --csm 1.2 -a full --ccj system-ccj.json  --sls-file sls_file.json --folder generated
+    ```
+
+## Compare the generated CSM 1.2 switch configurations with running configurations
+
+Compare the current running configuration with the generated configuration.
+
+Essentially this would be to backup current configuration to your workstation and comparing it against the CANU generated configuration.
+
+Example of CANU pulling configuration.
+
+```bash
```

Comment:
we're mixing and matching <dell/aruba/blah> tagging with physical example `--ip 192.168.1.1` for instance.  We should pick one.  Also it's not clear that `--ip` is the switch IP that matches with the generated config.  Having just been in this position someone will figure the <> tag stuff out and flat out paste in 192.168.1.1 and then file a bug because this failed.  

Not sure what the best solution is.